### ISSUE TITLE
Scrolling and pressing with touchpad not working on iPadOS 14.0, Safari.

### ISSUE THREAD

**Issue by leoleoasd** [cdr/code-server#2123]
- Web Browser: Safari
- Local OS: iPadOS 14.0
- Remote OS: Ubuntu 18.04
- Remote Architecture: and64
- `code-server --version`: 1.5.0

Scrolling and pressing with touchpad (magic keyboard) isn’t working. Haven’t tested with a mouse yet.

**Comment by snazzybunny** [comment]
Have the same issue. Is a fix on the way?

**Comment by code-asher** [comment]
I think this is a duplicate of https://github.com/cdr/code-server/issues/1455? Will close with that assumption but let me know if it seems different. There's no incoming fix at the moment, I think we're still waiting on a webkit fix last I recall.
